README.md

```diff
@@ -1,50 +1,49 @@
 |[DRM Fairplay](https://github.com/jwplayer/jwplayer-ios-bestPracticeApps/tree/main/JWBestPracticeApps/DRM%20Fairplay#drm-fairplay---best-practices-app)  | Simple implementation of JWPlayerKit SDK that plays protected video content through FairPlay (DRM). |
 |[Google IMA Ads](https://github.com/jwplayer/jwplayer-ios-bestPracticeApps/tree/main/JWBestPracticeApps/Google%20IMA%20Ads#google-ima-ads---best-practices-app) | Simple implementation of JWPlayerKit SDK with Google IMA. |
 |[Google DAI Ads](https://github.com/jwplayer/jwplayer-ios-bestPracticeApps/tree/main/JWBestPracticeApps/Google%20DAI%20Ads#google-dai-ads---best-practices-app) | Simple implementation of JWPlayerKit SDK with Google DAI. |
 |[JWPlayer Ads](https://github.com/jwplayer/jwplayer-ios-bestPracticeApps/tree/main/JWBestPracticeApps/JWPlayer%20Ads#jwplayer-ads---best-practices-app) | Simple implementation of JWPlayerKit SDK with advertising. |
 |[Recommendations](https://github.com/jwplayer/jwplayer-ios-bestPracticeApps/tree/main/JWBestPracticeApps/Recommendations) | Simple implementation including recommended related content. |
 
-## Setup
-
-Before compiling, this uses some pod dependencies so you will require [cocoapods](https://guides.cocoapods.org/using/getting-started.html) in you machine.
-
-Steps to compile:
-* Make sure your Xcode is closed.
-* In the terminal.app, in the project directory root.
-* Run:
-```console
-pod install
-```
-* This should update the dependencies in your Pods
-* Now open the `.xcworkspace`
+[^chromecastDisclaimer]: Unlike the other apps in this project which are *working* code examples, this app is a valid code example that nevertheless does **not** work as-is. 
+Rather, it requires the **dynamic** build of the Google Cast SDK. As Google does not distribute this via CocoaPods, it must be [added manually](https://developers.google.com/cast/docs/ios_sender#manual_setup) to a given project — as we do not include external frameworks in our repositories. Please refer to our [documentation](https://developer.jwplayer.com/jwplayer/docs/ios-enable-casting-to-chromecast-devices) for more information.
 
+## Setup
 
-Add your JWPlayer license key to the AppDelegate for the target project.
+| :information_source: The project uses [cocoapods](https://guides.cocoapods.org/using/getting-started.html) to manage dependencies. |
+|-|
 
-Here:
-```swift 
-JWPlayerKitLicense.setLicenseKey(<#Your License Key#>)
-```
+#### Steps to compile:
+1. Make sure Xcode is closed.
+1. In the Terminal app, navigate to the project's root directory.
+1. To download and integrate the dependencies in the project, run: 
+    ```console
+    pod install
+    ```
+1. Open `JWBestPracticeApps-4x.xcworkspace`.
+1. Add your [JWPlayer iOS SDK license key](https://support.jwplayer.com/articles/android-and-ios-sdk-reference#:~:text=HOW%20DO%20I%20GET%20THE%20IOS%20OR%20ANDROID%20SDK%3F%20HOW%20DO%20I%20USE%20THE%20SDKS%3F) to the AppDelegate for the target project:
```

Comment:
All steps are 1^

Reply:
@david-almaguer Since [the actual numbers used doesn't matter](https://daringfireball.net/projects/markdown/syntax#:~:text=It%E2%80%99s%20important%20to%20note%20that%20the%20actual%20numbers%20you%20use%20to%20mark%20the%20list%20have%20no%20effect%20on%20the%20HTML%20output%20Markdown%20produces.) (just that it _is_ a number), I chose to use `1.` for each of them so that it would stand out that it is an ordered list, and also if the items were later switched around, it would not be confusing. 

Check with the Preview to make sure the list appears as expected, but it should appear in order.

Reply:
Gotcha!